src/background/index.js

```diff
@@ -34,6 +66,8 @@ browser.runtime.onMessage.addListener(async function (request, sender) {
     return await handleNewRandomAlias(request.currentUrl);
   } else if (request.tag === "GET_APP_SETTINGS") {
     return await handleGetAppSettings();
+  } else if (request.tag === "EXTENSION_SETUP") {
+    return await handleExtensionSetup();
```

Comment:
Can you restrict this to simplelogin.io and staging? Just to avoid unintended side effects with other pages.